app/auth/callback/route.ts

```diff
@@ -2,7 +2,6 @@ import { NextResponse } from 'next/server';
 
 export async function GET(request: Request) {
   const requestUrl = new URL(request.url);
-  const code = requestUrl.searchParams.get('code');
```

Comment:
Why is this touched? I don't think it has anything to do with this PR does it?

Reply:
pnpm build failed because the `'code'` was not used anywhere so I deleted it.